src/foam/nanos/auth/User.js

```diff
@@ -493,6 +494,13 @@ foam.CLASS({
       createVisibility: 'HIDDEN',
       updateVisibility: 'RO',
       section: 'administrative'
+    },
+    {
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.auth.LifecycleState',
+      name: 'lifecycleState',
+      value: foam.nanos.auth.LifecycleState.PENDING,
+      visibility: 'HIDDEN'
```

Comment:
do we not want this permissioned for admins?

Reply:
Covereed in https://nanopay.atlassian.net/browse/NP-561